logging/unit_tests/test_client.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import mock
```

Comment:
1. Please import this at runtime (per our "testing philosophy")
1. Even if you weren't going to import it at runtime, you've broken the "stdlib --> 3rd party --> local package" rule for imports (`mock` is 3rd party, `unittest` is stdlib)